pkg/reconciler/route/resources/service_test.go

```diff
@@ -127,7 +127,7 @@ func TestNewMakeK8SService(t *testing.T) {
 		route: r,
 		ingress: &netv1alpha1.Ingress{
 			Status: netv1alpha1.IngressStatus{
-				LoadBalancer: &netv1alpha1.LoadBalancerStatus{
+				DeprecatedLoadBalancer: &netv1alpha1.LoadBalancerStatus{
```

Comment:
Should we just drop them?

Reply:
Not sure. I'm just fixing the build :-)

Reply:
But I think it might break build or miss coverage until we remove fields